core/server/services/auth/session/middleware.js

```diff
@@ -39,7 +39,7 @@ const getSession = (req, res, next) => {
             cookie: {
                 maxAge: constants.SIX_MONTH_MS,
                 httpOnly: true,
-                path: '/ghost',
+                path: urlService.utils.getSubdir() + '/ghost',
```

Comment:
`urlService.utils.urlFor('admin')`

Reply:
That's gonna include a trailing slash, which means the cookie won't be sent if we're on a page without one - I think that would be confusing behaviour :confused: I could use `urlFor` and strip the slash?

Reply:
Hmm, but Ghost forces the trailing slash for any page visit

Reply:
Leave this as is, if you think it's safer

Reply:
Just decide and selfe merge 👍

Reply:
> Hmm, but Ghost forces the trailing slash for any page visit

It does, I'm just worried that we could be bitten in the future :shrimp: I think this way is slightly safer - but also it's not as nice